File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/hyperbolic/TanhZ.java
package net.fwitz.math.main.binary.complex.functions.hyperbolic;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class TanhZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("tanh(z)")
                .computeFn(Complex::tanh)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/hyperbolic/TanhZ.kt
package net.fwitz.math.main.binary.complex.functions.hyperbolic

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object TanhZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("tanh(z)")
        .computeFn(Complex::tanh)
        .render()
}